src/kkt.c

```diff
@@ -184,9 +184,9 @@ static PyGetSetDef KKT_getset[] = {
    "Character representing the quality of primal feasibility.\n"
    "'H', high, 'M', medium, 'L', low, or '?' wrong or infeasible."},
 
-  {"de_ae_row", (getter)KKT_de_ae_col, (setter)NULL,
+  {"de_ae_col", (getter)KKT_de_ae_col, (setter)NULL,
```

Comment:
Good point. I changed those attribute names back, and fixed the tests accordingly. 

I'll submit a new PR with the name changes after I comb through and figure out what parts of the documentation need to be updated.